staging/src/k8s.io/api/admissionregistration/v1/types.go

```diff
@@ -307,6 +307,18 @@ type ValidatingWebhook struct {
 	// include any versions known to the API Server, calls to the webhook will fail
 	// and be subject to the failure policy.
 	AdmissionReviewVersions []string `json:"admissionReviewVersions" protobuf:"bytes,8,rep,name=admissionReviewVersions"`
+
+	// MatchConditions is a list of conditions on the AdmissionRequest ('request') that must be met
+	// for a request to be sent to this webhook. All conditions in the list must evaluate to TRUE for
+	// the request to be matched. This is an alpha feature and managed by the AdmissionWebhookMatchConditions feature gate.
+	// If this field is not set no matching will occur through cel. If any condition in the list fails to be met apiserver
```

Comment:
```
// If this field is not set no matching will occur through cel.
```

"no matching will occur" sounds like the webhook won't be called. Maybe something like: `An empty list of matchConditions matches all requests.`